### PR TITLE
375 add missing FHIR resources to consolidated

### PR DESCRIPTION
**Pull request by leite08**
Ref. metriport/metriport#375

### Dependencies

none

### Description

Add missing FHIR resources to `/consolidated` endpoint

### Release Plan

- nothing special

### CODE REVIEW

File: api/app/src/command/medical/patient/consolidate-data.ts [hunk 1]
```diff
@@ -154,4 +175,11 @@ export const resourceTypeForConsolidation = [
   "Specimen",
 ] as const;
 
+export const resourcesSearchableBySubject = ["AdverseEvent", "RiskAssessment", "Task"] as const;
```
Comment (Goncharo) [inline api/app/src/command/medical/patient/consolidate-data.ts:178]: `AdverseEvent` and `Task` need to be added to the [docs](https://docs.metriport.com/medical-api/api-reference/patient/get-patient-consolidated)

**Review by Goncharo — CHANGES_REQUESTED** [review]
Code LGTM - just missing the docs update.

Also, not sure why this PR is touching DAPI code, may need to update the branch?

**Review by Orta21 — APPROVED** [review]

File: api/app/src/command/medical/patient/consolidate-data.ts (hunk 1 above)
Comment (leite08) [reply api/app/src/command/medical/patient/consolidate-data.ts:178]: Good catch!

**Comment by leite08** [comment]
@Goncharo updated docs and rebased from `develop`, thanks!

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**